js/src/dapps/registry/Lookup/lookup.js

```diff
@@ -11,41 +13,50 @@ export default class Lookup extends Component {
     lookup: PropTypes.object
   }
 
-  state = { name: '', key: '' };
+  state = { name: '', key: 'A' };
 
   render () {
-    const self = this;
-    const onNameChange = (e) => {
-      self.setState({ name: e.target.value });
-    };
-    const onKeyChange = (e) => {
-      self.setState({ key: e.target.value });
-    };
-    const onLookupClick = () => {
-      self.props.actions.lookup(self.state.name, self.state.key);
-    };
-
+    const { name, key } = this.state;
+    const props = this.props.lookup;
```

Comment:
Look at Tomasz' comment above, you can do `<Component {...this.props} />` to pass everything available.